### PR TITLE
Dice

### PR DESCRIPTION
**Pull request by behweifeng**

### CODE REVIEW

File: client/src/components/DiceApp.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import Dice from "react-dice-roll";
+
+export default function DiceApp({ handlerKey }: { handlerKey: string }) {
```
Comment (jay-ng-mc) [inline client/src/components/DiceApp.tsx]: What's the purpose of handlerKey?

File: client/src/components/DiceRoller.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {ThemeProvider, theme, CSSReset, Flex, Box, Heading, IconButton, Link, FormControl, FormLabel, Input, Stack, Checkbox, Button} from "@chakra-ui/react";
```
Comment (jay-ng-mc) [inline client/src/components/DiceRoller.tsx]: It seems like the following classes are not used: IconButton, Link, FormControl, FormLabel, Input, Stack, Checkbox, Button

Are they necessary?

**Review by jay-ng-mc — CHANGES_REQUESTED** [review]

**Review by jay-ng-mc — APPROVED** [review]

File: client/src/components/DiceApp.tsx (hunk 1 above)
Comment (behweifeng) [reply client/src/components/DiceApp.tsx]: Removed for now

File: client/src/components/DiceRoller.tsx (hunk 2 above)
Comment (behweifeng) [reply client/src/components/DiceRoller.tsx]: Not necessary as they are not used, removed

**Pull request merged**